consensus/poet/families/tests/validator_reg_message_factory.py

```diff
@@ -287,3 +289,34 @@ def create_set_request_validator_map(self):
     def create_set_response_validator_map(self):
         addresses = [self._key_to_address("validator_map")]
         return self._factory.create_set_response(addresses)
+
+    def create_get_request_report_key_pem(self):
+        return \
+            self._factory.create_get_request(
+                ['000000a87cb5eafdcca6a87ccc804f5546a'
+                 'b8e97a7d614626e4500e3b0c44298fc1c14'])
+
+    def create_get_response_report_key_pem(self, pem=None):
+        setting = Setting()
+        if pem is not None:
+            entry = Setting.Entry(key='sawtooth.poet.report_public_key_pem',
+                                  value=pem)
+            setting.entries.extend([entry])
+
+        data = setting.SerializeToString()
+        return self._factory.create_get_response(
+            {'000000a87cb5eafdcca6a87ccc804f5546a'
+             'b8e97a7d614626e4500e3b0c44298fc1c14': data})
+
+    def create_get_response_simulator_report_key_pem(self, pem=None):
+        pem = '''-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArMvzZi8GT+lI9KeZiInn
```

Comment:
The indentation is because it is a multi-line string and if I indent, it will insert spaces after the newline causing the PEM to not parse correctly into a valid public key.